lib/CXGN/BrAPI/v2/Locations.pm

```diff
@@ -192,9 +192,10 @@ sub store {
     my $schema = $self->bcs_schema();
     my @location_ids;
 
-	if (!$user_id) {
-		return CXGN::BrAPI::JSONResponse->return_error($self->status, sprintf('You must login and have permission to access this BrAPI call.'));
-	}
+	# TODO: removed check for use with bi-api for now without token
+	#if (!$user_id) {
+	#	return CXGN::BrAPI::JSONResponse->return_error($self->status, sprintf('You must login and have permission to access this BrAPI call.'));
+	#}
```

Comment:
I think it could just be one card to add to bi-api and update breedbase just so one doesn't get lost. We can always split it apart when we actually work on it. Created card: https://breedinginsight.atlassian.net/browse/BI-707?atlOrigin=eyJpIjoiYWIzMzQ2YjUyNjFhNGQyMGJkOWVjMzE2ZjVhM2ExNjIiLCJwIjoiaiJ9